clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -6409,6 +6409,60 @@ static void handleRequiresCapabilityAttr(Sema &S, Decl *D,
   D->addAttr(RCA);
 }
 
+static void handleCxx26AnnotationAttr(Sema &S, Decl *D, const ParsedAttr &AL) {
+  Expr *CE = AL.getArgAsExpr(0);
+  if (CE->isLValue()) {
```

Comment:
Can you include the standardeeze here as to what exactly is happening/the rules of what should happen on the RHS here?